### PR TITLE
DDF-3044 removed XacmlConstants.java

### PR DESCRIPTION
**Pull request by shaundmorris**
#### What does this PR do?
Addresses technical debt to remove a constants class

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Security](https://github.com/orgs/codice/teams/security)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@stustison 
@lessarderic 

#### How should this be tested? (List steps with links to updated documentation)
Build passes

#### What are the relevant tickets?
DDF-3044
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

---
This change is [Reviewable](https://reviewable.io/reviews/codice/ddf/2024)

### CODE REVIEW

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

File: platform/security/pdp/security-pdp-authzrealm/src/test/java/ddf/security/pdp/realm/xacml/XacmlPdpTest.java [hunk 1]
```diff
@@ -203,77 +232,77 @@ public void testResourceIsNotPermitted() {
 
     @Test
     public void testParseAttributeTypeBoolean() {
-        assertTrue(XACMLConstants.BOOLEAN_DATA_TYPE.equals(testRealm.getXacmlDataType("true")));
+        assertTrue(BOOLEAN_DATA_TYPE.equals(testRealm.getXacmlDataType("true")));
```
Comment (lessarderic) [inline platform/security/pdp/security-pdp-authzrealm/src/test/java/ddf/security/pdp/realm/xacml/XacmlPdpTest.java]: ❓ Should we replace those `assertTrue` with `assertThat(..., is(...))` while we're at it?

Comment (shaundmorris) [inline platform/security/pdp/security-pdp-authzrealm/src/test/java/ddf/security/pdp/realm/xacml/XacmlPdpTest.java]: sure @lessarderic when you get a chance can you take a quick look?

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Comment by shaundmorris** [comment]
build now

**Pull request merged**